backend/api/Database/Models/Inspection.cs

```diff
@@ -50,10 +50,12 @@ public Inspection(Inspection copy, InspectionStatus? inspectionStatus = null)
         [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
         public string Id { get; set; }
 
+        [Required]
         [MaxLength(200)]
         // ReSharper disable once AutoPropertyCanBeMadeGetOnly.Local
         public string? IsarStepId { get; private set; } = Guid.NewGuid().ToString();
 
+        [Required]
```

Comment:
Should this be required? For instance if you want to take a sound recording at a given spot, you dont need an InspectionTarget

Reply:
I tried to make it match the nullability of the C# object, but I can make them both nullable

Reply:
Think we make both nullable then?
`public Position? InspectionTarget`